datasetinsights/datasets/synthetic.py

```diff
@@ -264,8 +264,8 @@ def _preprocess_dataset(data_path):
 
         # This is for dataset generated by unity simulation.
         # In this case, all data are downloaded directly in the data_path
-        if SynDetection2D.is_dataset_files_present(data_path):
-            return data_path
+        # if SynDetection2D.is_dataset_files_present(data_path):
```

Comment:
This breaks the case where a synthetic dataset is downloaded from Unity Simulation